### PR TITLE
Making pkg_check return error code.

### PR DESCRIPTION
**Pull request by hklai**

### CODE REVIEW

File: toolbox/pkg_check/main.go [hunk 1]
```diff
@@ -102,11 +104,13 @@ func printDelta(deltas, report, baseline map[string]float64) {
 	for pkg, delta := range deltas {
 		if delta+*threshold < 0 {
 			glog.Errorf("Coverage dropped: %s:%f%% (%f%% to %f%%)", pkg, delta, baseline[pkg], report[pkg])
+			code = 2
```
Comment (rkpagadala) [inline toolbox/pkg_check/main.go]: add a constant for return code.
otherwise lgtm

Comment (hklai) [reply toolbox/pkg_check/main.go]: Done

File: toolbox/pkg_check/main_test.go [hunk 2]
```diff
@@ -78,6 +78,62 @@ func TestFindDelta(t *testing.T) {
...
+			"P4": 90,
+		},
+		// baseline
+		map[string]float64{
+			"P1": 50,
+			"P2": 60,
+			"P3": 100,
+			"P5": 60,
+		})
+	if code != 2 {
```
Comment (rkpagadala) [inline toolbox/pkg_check/main_test.go]: sorry this one too

Comment (hklai) [reply toolbox/pkg_check/main_test.go]: Done

**Pull request merged**